backend/stats/views.py

```diff
@@ -1,3 +1,28 @@
 from django.shortcuts import render
+from rest_framework.views import APIView
+import plotly.graph_objects as go
+import plotly.io as pio
+from django.http import HttpResponse
+from rest_framework.response import Response
+from rest_framework import status
+# from django.utils.decorators import method_decorator
+# from User.serializers import UserOutputSerializer
 
-# Create your views here.
+# from shared_utilities.decorators import must_be_authenticated, \
+# 									must_be_body_user_id, \
+# 									valid_serializer_in_body
+
+class StatsView(APIView):
```

Comment:
Could and should be added later but will need the decorators. No need to add them now unless we actually start getting some data to build the stats from.